bootstrap.py

```diff
@@ -495,6 +495,22 @@ def disassociate_host(host_id):
     print_running("Disassociating host id %s for host %s" % (host_id, FQDN))
     put_json(myurl)
 
+def configure_subscription_manager():
+    configparser = SafeConfigParser()
+    configparser.read('/etc/yum/pluginconf.d/product-id.conf')
+    if configparser.get('main','enabled') == '0':
+      print_generic("Product-id yum plugin was disabled. Enabling...")
+      configparser.set('main', 'enabled', '1')
+      with open('/etc/yum/pluginconf.d/product-id.conf','w') as configfile:
```

Comment:
Will have for a long time, i can ensure... 
I know that there are still RHEL4 and RHEL3 in prod.